quiz/backend/mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+python_version = 3.11
+ignore_missing_imports = True
```

Comment:
możesz to wyjaśnić?

Reply:
jak zingorujesz błędy, to nie ma błędów? :)  Chodzi o 3rd party libraries, bez tego będzie się rzucał, jak jakaś biblioteka nie ma stubs